theia-ide/theia-apps
# [java-docker] use builtin and external ext

#### What it does


Update the `theia-java-docker:next` image to include builtins and external VS Code extensions

#### How to test
Build and verify the changes (that plugins are downloaded and picked up when started)

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)


Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>
